lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -327,6 +323,74 @@ void setWantConsolidation(MHWRender::MRenderItem& renderItem, bool state)
     renderItem.setWantSubSceneConsolidation(state);
 #endif
 }
+
+PrimvarInfo* _getInfo(const PrimvarInfoMap& infoMap, const TfToken& token)
```

Comment:
I eliminated the special data members to specific buffers. In their place I search for the buffers in the map when necessary.

Reply:
I think it should be fine, but it would be nice to confirm whether it can impact performance on animated assets.

Reply:
I haven't seen a performance impact on the animation scenes we have.